test/specs/mainnet/01_TransferringFunds/Arbitrum.spec.js

```diff
@@ -77,7 +77,9 @@ describe('The PrimeSDK, when transfer a token with arbitrum network on the MainN
       process.env.PROJECT_KEY,
```

Comment:
Can you pls add try/catch block to DataUtils initialisation to all files and display appropriate msg if it errors out

Reply:
I have modified the code and commit it.